lib/linguist/languages.yml

```diff
@@ -4110,6 +4110,8 @@ Shell:
   - ".command"
   - ".fcgi"
   - ".ksh"
+  - ".mksh"
+  - ".pdksh"
```

Comment:
I'm not sure there are enough [`.mksh`](https://github.com/search?utf8=%E2%9C%93&q=extension%3Amksh+NOT+kdjskfjdjf&type=) or [`.pdksh`](https://github.com/search?utf8=%E2%9C%93&q=extension%3Apdksh+NOT+dkfjdkfkd&type=) files on github.com to warrant their addition as file extensions. Adding the shebang might be sufficient anyway...